pkg/webhook/server/server.go

```diff
@@ -170,15 +170,18 @@ func (m *podInjector) Handle(ctx context.Context, req admission.Request) admissi
 	}
 
 	useImmutableImage := ""
+	imagePullPolicy := corev1.PullIfNotPresent
 	if oa.Status.UseImmutableImage {
 		useImmutableImage = "true"
+		imagePullPolicy = corev1.PullAlways
 	}
```

Comment:
Always set it to PullAlways, there is no reason not to. If you don't use the immutable image, then we use the Operator image, and we want to use the same one used by the Operator Pod.

Reply:
Done